translations/en_GB/atrium-translations-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionPathAssertion.kt

```diff
@@ -33,5 +33,7 @@ enum class DescriptionPathAssertion(override val value: String) : StringBasedTra
     FAILURE_DUE_TO_WRONG_FILE_TYPE("was %s instead of %s"),
     FAILURE_DUE_TO_LINK_LOOP("found a symbolic link loop: %s"),
     HINT_CLOSEST_EXISTING_PARENT_DIRECTORY("the closest existing parent directory is %s"),
-    HINT_FOLLOWED_SYMBOLIC_LINK("followed the symbolic link %s to %s")
+    HINT_FOLLOWED_SYMBOLIC_LINK("followed the symbolic link %s to %s"),
+    HAS_SAME_TEXTUAL_CONTENT("has same textual content"),
```

Comment:
please add the following to ch.tutteli.atrium.translations.DescriptionPathAssertion:
```
HAS_SAME_TEXTUAL_CONTENT("hat denselben textlichen Inhalt"),
HAS_SAME_BINARY_CONTENT("hat denselben binären Inhalt")
```